lib/game.rb

```diff
@@ -0,0 +1,80 @@
+class Game 
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]
+  ]
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def board
+    @board
+  end
+
+  def current_player
+    board.turn_count.even? ? player_1 : player_2
+  end
+
+  def over?
+    draw? || won?
+  end
+
+  def won?
+    WIN_COMBINATIONS.detect do |combo|
+     combo.all? {|position| board.cells[position] == "X"} || combo.all? {|position| board.cells[position] == "O"}
+    end
+  end
```

Comment:
This is one of the simpler implementations of this method I've seen. Very clear, nice job.